engines/ospray/OSPRayModel.h

```diff
@@ -31,28 +30,47 @@ namespace brayns
 class OSPRayModel : public Model
 {
 public:
-    OSPRayModel() = default;
+    OSPRayModel(AnimationParameters& animationParameters,
+                VolumeParameters& volumeParameters);
     ~OSPRayModel() final;
 
     void setMemoryFlags(const size_t memoryManagementFlags);
 
-    void commit() final;
+    void commitGeometry() final;
+    bool commitTransferFunction();
 
     OSPModel getModel() const { return _model; }
     OSPModel getBoundingBoxModel() const { return _boundingBoxModel; }
     OSPModel getSimulationModel() const { return _simulationModel; }
     MaterialPtr createMaterial(const size_t materialId,
                                const std::string& name) final;
+    SharedDataVolumePtr createSharedDataVolume(const Vector3ui& dimensions,
+                                               const Vector3f& spacing,
+                                               const DataType type) const final;
+    BrickedVolumePtr createBrickedVolume(const Vector3ui& dimensions,
+                                         const Vector3f& spacing,
+                                         const DataType type) const final;
```

Comment:
Why did you move this functions from the scene here?

Reply:
because the volumes need a reference to a transfer function, and this is per model now. Leaving this function here would need a modelID as well or the transfer function itself that has to be used for the new volume.